src/dmd/dmodule.d

```diff
@@ -370,12 +370,10 @@ extern (C++) final class Module : Package
     int searchCacheFlags;       // cached flags
 
     /**
-     * A root module is one that will be compiled all the way to
-     * object code.  This field holds the root module that caused
-     * this module to be loaded.  If this module is a root module,
-     * then it will be set to `this`.  This is used to determine
-     * ownership of template instantiation.
-     */
+    This field holds the root module that caused this module
+    to be loaded.  If this module is a root module, then it will
+    be set to `this`.
+    */
```

Comment:
Unless it's no longer true, I'd keep the part of the comment that says "This is used to determine the ownership of templaet instantiation".

Reply:
The information was just moved to the `isRoot` function.